ui-app/client/src/components/Otp/OtpStyle.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { processStyleDefinition } from '../../util/styleProcessor';
+import { styleProperties, styleDefaults } from './otpStyleProperties'
+
+const PREFIX = '.comp.compOtp';
+export default function OtpStyle({ theme }: { theme: Map<string, Map<string, string>> }) {
+	const css =
+		`
+		${PREFIX} {
+			display: flex;
+			
+		}
+
+		${PREFIX} ._inputBox {
+			width: 30px; 
+			height: 30px; 
+			margin: 8px;
+			font-size: 16px; 
+			text-align: center;
+			outline: none; 
+			border-style:solid;
```

Comment:
should be in styleProperties if there is not an specific reason for it to be here.